### ISSUE TITLE
Auto Logout

### ISSUE THREAD

**Issue by Surendrajat** [ThirtyDegreesRay/OpenHub#88]
Hi!
Thanks for creating this useful app. I've been using this app for a while now and sometime I face this issue of auto logout. It happens mainly when I open app while phone's offline or connected to wifi with proxy server enabled.
It is not very frequent and can be ignored but It'll be nice if gets fixed.

**Comment by ThirtyDegreesRay** [comment]
Hi, thanks for your feedback, I will fix it.

**Comment by anonymous-user** [comment]
Any update on this? I find it rather annoying that I keep having to re-login to the app.

**Comment by matthew-nm** [comment]
I can confirm this issue. It seems that whenever I open the app after having not used it in a while, I have to reauthorize with OAuth.

**Comment by ThaChillera** [comment]
Yeah this issue is very annoying, makes me want to uninstall (since all github links open with openhub). ETA on a fix?

**Comment by matthew-nm** [comment]
@ThaChillera The GitHub official app is pretty good. I'm using that now.
